subscription-manager.spec

```diff
@@ -290,6 +302,61 @@ the remote in the currently deployed .origin file.
 %endif
 
 
+%package -n python-rhsm
```

Comment:
Have we ruled out renaming the package to `subscription-manager-rhsm` or the like and using the `Obsoletes` and `Provides` directives in the package header?

Reply:
> Have we ruled out renaming the package to subscription-manager-rhsm or the like and using the Obsoletes and Provides directives in the package header?

No, we have not ruled this out. What are your thoughts on benefits of doing that?

Reply:
I think the main benefit is it allows people to find the source.  If I see a package installed on my system named "python-rhsm", I'm going to try running `fedpkg clone python-rhsm`.  That won't work and then I have to go to Koji and figure out where the heck "python-rhsm" comes from.  In my opinion, renaming the package to "subscription-manager-rhsm" makes it apparent where the code is coming from.  It's a nice-to-have I think.

Renaming is possible.  Fedora has instructions [here](https://fedoraproject.org/wiki/Packaging:Guidelines#Renaming.2FReplacing_Existing_Packages).

Reply:
I like this renaming too! 👍